#!/usr/bin/python3


"""Raises a name exception with the provided message."""
raise NameError(message)
